inventory/idm/group_vars/idm.yml

```diff
@@ -0,0 +1,27 @@
+---
+ansible_user: cloud-user
+ansible_become: True
+idm_master_hostname: idm.example.com
+idm_domain: example.com
+idm_realm: example.com
+idm_dm_password: my_dm_password
+idm_admin_password: my_admin_password
+rhsm_username: my_rhsm_username
+rhsm_password: my_rhsm_password
+rhsm_pool: my_pool_id
+rhsm_repos:
+ - rhel-7-server-rpms
+ - rhel-7-server-extras-rpms
```

Comment:
Don't think the `extras` repo is needed for this one - at least we don't have it in our internal inventory used for our DC.